dataset/dictionary/organizations.ts

```diff
@@ -83,7 +83,15 @@ export default [
     pronunciationJa: "まじょかい",
     tags: [ "mondstadt", "organization" ],
   },
-
+  {
+    en: "Sal Vindagnyr",
+    ja: "シャール・フィンドニール",
+    zhCN: "沙尔·芬德尼尔",
+    zhTW: "沙爾·芬德尼爾",
+    notes: "ドラゴンスパインの位置にかつて存在していた王国",
+    tags: [ "mondstadt", "dragonspine", "organization" ],
```

Comment:
There is no best tag for the nations in the Teyvat history now, but considering the consistency with the existing nations (Natlan, Fontaine, etc.), `location` tag might be a better fit.

```suggestion
    tags: [ "mondstadt", "dragonspine", "location" ],
```